website/docs/Examples/Integrate - Spark.md

```diff
@@ -0,0 +1,113 @@
+# Integrate Spark for Distributed Training
+
+FLAML has integrated Spark for distributed training. There are two main aspects of integration with Spark:
+- Use Spark ML estimators for AutoML.
+- Use Spark to run training in parallel spark jobs. 
+
+## Spark ML Estimators
+
+FLAML integrates estimators based on Spark ML models. These models are trained in parallel using Spark, so we called them Spark estimators. To use these models, you first need to organize your data in the required format.
+
+### Data
+
+For Spark estimators, AutoML only consumes Spark data. FLAML provides a convenient function `to_pandas_on_spark` in the `flaml.automl.spark.utils` module to convert your data into a pandas-on-spark (`pyspark.pandas`) dataframe/series, which Spark estimators require.
+
+This utility function takes data in the form of a `pandas.Dataframe` or `pyspark.sql.Dataframe` and converts it into a pandas-on-spark dataframe. It also takes `pandas.Series` or `pyspark.sql.Dataframe` and converts it into a pandas-on-spark series. If you pass in a `pyspark.pandas.Dataframe`, it will not make any changes, so don't worry about multiple calls with the same data.
+
+This function also accepts optional arguments `index_col` and `default_index_type`. 
+- `index_col` is the column name to use as the index, default is None.
+- `default_index_type` is the default index type, default is "distributed-sequence".
```

Comment:
I'd link to the official Spark documentation on default index type: https://spark.apache.org/docs/latest/api/python/user_guide/pandas_on_spark/options.html#default-index-type

Reply:
sure will do